### ISSUE TITLE
Using private handleEvent API

### ISSUE THREAD

**Issue by mandarin6b0** [CombineCommunity/CombineCocoa#80]
**Describe the bug**
I get an error while trying to upload a binary to Testflight
```
The app references non-public selectors in Payload/XXX.app/Frameworks/CombineCocoa.framework/CombineCocoa: handleEvent
```

**To Reproduce**
1. Include CombineCocoa (using cocoapods in my case)
2. Archive and upload to Testflight

**Expected behavior**
No error when distributing

**Device:**
- XCode 14.0.1
- CombineCocoa version 0.4.0

**Comment by StefanNestorov** [comment]
I have the exact same issue

**Comment by freak4pc** [comment]
That's really interesting. handleEvent is the name of [this method](https://github.com/CombineCommunity/CombineCocoa/blob/main/Sources/CombineCocoa/CombineControlProperty.swift#L90-L94).

Could it be that Apple is confusing it with some private method? 
@mandarin6b0 @StefanNestorov Will you be able testing a branch if I give you one?

**Comment by mandarin6b0** [comment]
@freak4pc sure

**Comment by freak4pc** [comment]
Can you try the branch `handleevent-change` please? @mandarin6b0

**Comment by mandarin6b0** [comment]
@freak4pc I still get the problem. I found one more place with [handleEvent](https://github.com/CombineCommunity/CombineCocoa/blob/handleevent-change/Sources/CombineCocoa/CombineControlEvent.swift#L69). Should it be changed too?

**Comment by freak4pc** [comment]
Changed and apparently didn’t commit. Try now.

**Comment by mandarin6b0** [comment]
> Changed and apparently didn’t commit. Try now.
> […](#)
> On 20 Oct 2022, 12:23 +0300, Tien Nguyen ***@***.***>, wrote: @freak4pc I still get the problem. I found one more place with handleEvent. Should it be changed too? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

It's working now 👍

**Comment by freak4pc** [comment]
I'll merge it. THanks!

**Comment by freak4pc** [comment]
Released in 0.4.1

**Comment by mandarin6b0** [comment]
Thank you!On 20 Oct 2022, at 18:46, Shai Mishali ***@***.***> wrote:﻿
Released in 0.4.1
